lms/js_config_types.py

```diff
@@ -13,6 +13,11 @@ class AnnotationMetrics(TypedDict):
     last_activity: str | None
 
 
+class CourseMetrics(TypedDict):
+    assignment_count: int
```

Comment:
I just realized we have used `annotations: int` and `replies: int` for annotation metrics, so perhaps this should be `assignments: int` rather than `assignment_count: int`?

Reply:
Yes, good point